src/components/ConnectForm.vue

```diff
@@ -272,7 +281,7 @@ export default {
 
 		showErrors(error) {
 			// ToDo: Show Modal with Logs component?
-			alert(`Close this popup and press the 'F12' key to open the Browser console for more details.`);
+			alert(`Close this popup and press the 'F12' key (Chrome) or 'Ctrl+Shift+K' (Firefox) to open the Browser console for more details.`);
```

Comment:
F12 on FF/Win also works. Is this CTRL + Shift + L a Linux thing?

Reply:
Nevermind, this has been replaced by a modal and will be removed when merging...

Reply:
Oh this is new to me. For Firefox it has always been Ctrl+Shift+K (also on Windows) but it seems they obeyed the new hotkey standard Chrome introduced: F12 also works for me, I just tried.